dffml/model/model.py

```diff
@@ -50,13 +51,20 @@ async def train(self, sources: Sources):
         """
         raise NotImplementedError()
 
-    @abc.abstractmethod
-    async def accuracy(self, sources: Sources) -> Accuracy:
+    async def accuracy(
+        self, sources: Sources, accuracy_scorer: AccuracyContext
+    ) -> Accuracy:
         """
         Evaluates the accuracy of our model after training using the input records
         as test data.
         """
-        raise NotImplementedError()
+        if not hasattr(self.config, "predict") or not hasattr(
+            self.config, "features"
```

Comment:
I don't think we need the `or features` here. Since we don't use it in this function

Reply:
ok.